src/index.html

```diff
@@ -7,8 +7,50 @@
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Moyo header</title>
     <link rel="stylesheet" href="./style.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com"
+      crossorigin="true">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:ital,wght@0,100;0,300;0,400;0,500;0,700;0,900;1,100;1,300;1,400;1,500;1,700;1,900&display=swap"
+      rel="stylesheet">
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <a href="#">
+        <img class="logo" src="images/logo.png"
+        alt="logo" width="40"
+        height="40">
```

Comment:
Add better description to `alt`. Also formatting, check this [list - Keep your attributes correctly formatted
](https://github.com/mate-academy/layout_moyo-header/blob/master/checklist.md)
```suggestion
        <img 
          class="logo" 
          src="images/logo.png"
          alt="Moyo logo" 
          width="40"
          height="40"
        >
```